workspace.Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM node:19-bullseye-slim
+FROM node:20-bullseye-slim
 
 RUN apt-get update && apt-get install -y \
     git \
     python3 \
     make \
     g++
 
```

Comment:
This can be removed, the packages were needed for truffle installation IIRC.

Reply:
Done